whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisSearchServiceTest.java

```diff
@@ -52,6 +52,7 @@ public void search_disallowedFlags() {
                         Sets.newHashSet("TEST"),
                         "AARDVARK-MNT",
                         Collections.EMPTY_SET,
+                        null,
```

Comment:
Move the new argument to the end?

Reply:
Actually order is same as with Get method. So all query params are in together